test/suite.bats

```diff
@@ -40,6 +40,12 @@
 }
 
 
+@test "rclone is installed" {
+  run docker run --rm --entrypoint sh $IMAGE -c 'which rclone'
+  [ "$status" -eq 0 ]
+}
+
```

Comment:
@alnikyur why haven't you add `rclone run ok` test? It's vital to ensure that rclone is installed correctly.